InternshipProject#2/Manager/ICommentManager.cs

```diff
@@ -0,0 +1,14 @@
+﻿using InternshipProject_2.Models;
+using RequestResponseModels.Comment.Request;
+
+namespace InternshipProject_2.Manager
+{
+    public interface ICommentManager
+    {
+        //public Task<IEnumerable<Comment>> GetComments(int TicketId);
```

Comment:
Why are the methods commented out?

Reply:
Wanted to make a branch only using the Create Method. Just copied from the master branch the interface and didn't want to delete the other methods so I know what should I do next.